config/config.yml

```diff
@@ -3,4 +3,11 @@ backend: rpc://
 montagu_url: http://localhost:8080
 montagu_user: test.user@example.com
 montagu_password: password
-orderlyweb_url: http://localhost:8888
\ No newline at end of file
+orderlyweb_url: http://localhost:8888
+diagnostic_reports:
```

Comment:
Thinking about how the configuration would grow, do you think we might want to add a bit of structure here? The key `diagnostic_reports` feels different to the the server configuration above. Grouping these into:

```
servers:
  montagu:
    url: ...
    user: ...
  orderlyweb:
    url: ...
tasks:
  diagnostic_reports:
    ...
```

might make this grow more painlessly in the future